PubNub/Network/Parsers/PNSubscribeParser.m

```diff
@@ -188,36 +177,27 @@ + (NSDictionary *)parsedServiceResponse:(id)response withData:(NSDictionary *)ad
         }
         processedResponse = @{@"events":feedEvents,@"timetoken":timeToken};
     }
-    
+
     return processedResponse;
 }
 
 
 #pragma mark - Events processing
 
-+ (NSMutableDictionary *)eventFromData:(id)data forChannel:(NSString *)channel
-                                 group:(NSString *)group
-              withAdditionalParserData:(NSDictionary *)additionalData {
++ (NSMutableDictionary *)eventFromData:(id)data withAdditionalParserData:(NSDictionary *)additionalData {
     
     NSMutableDictionary *event = [NSMutableDictionary new];
-    if ([channel length]) {
-        
-        event[(![group length] ? @"subscribedChannel": @"actualChannel")] = channel;
-    }
-    if ([group length]) {
-        
-        event[@"subscribedChannel"] = group;
-    }
-    
+    NSString *channel = data[kPNMessageChannelKey];
+    NSString *subscriptionMatch = data[kPNMessageSubscriptionMatchKey];
+    event[@"subscribedChannel"] = subscriptionMatch;
```

Comment:
@jzucker2 here should be another assignment: _(subscriptionMatch?: channel)_ - there is a cases when subscription match will be empty (for example if client subscribed only on one channel and there is no multiplexing used).